src/Compilers/VisualBasic/Portable/Binding/Binder_Symbols.vb

```diff
@@ -341,7 +342,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         Dim diagName = GetBaseNamesForDiagnostic(typeSyntax)
                         diagInfo = NotFound(typeSyntax, diagName, binder, diagBag, reportedAnError)
 
-                        Return Binder.GetErrorSymbol(diagName, diagInfo)
+                        Return binder.GetErrorSymbol(diagName, diagInfo)
```

Comment:
GetErrorSymbol is an instance  member, so 'binder' is correct since that is the name of the locals. I do not think I changed this. I think it was just pretty-listed to the right casing

---
In reply to: [71442850](https://github.com/dotnet/roslyn/pull/12596#discussion_r71442850) [](ancestors = 71442850)